MVP/Resources/Translations.tr.resx

```diff
@@ -0,0 +1,465 @@
+    <value>Katkılar</value>
+  </data>
+  <data name="field_meetings" xml:space="preserve">
+    <value>Toplantılar</value>
+  </data>
+  <data name="field_members" xml:space="preserve">
+    <value>Üyeler</value>
+  </data>
+  <data name="field_number_of_attendees" xml:space="preserve">
+    <value>Katılımcı sayısı</value>
+  </data>
+  <data name="field_annual_quantity" xml:space="preserve">
+    <value>Yıllık miktar</value>
+  </data>
+  <data name="field_annual_reach" xml:space="preserve">
+    <value>Yıllık erişim</value>
+  </data>
+  <data name="field_number_of_events_participated" xml:space="preserve">
+    <value>Katılınan Etkinlik sayısı</value>
+  </data>
+  <data name="field_number_of_feedbacks_provided" xml:space="preserve">
+    <value>Sağlanan geri dönüş adedi</value>
+  </data>
+  <data name="field_number_of_samples" xml:space="preserve">
+    <value>Örnek sayısı</value>
+  </data>
+  <data name="field_number_of_downloads" xml:space="preserve">
+    <value>İndirme adedi</value>
+  </data>
+  <data name="field_number_of_sites" xml:space="preserve">
+    <value>Site sayısı</value>
+  </data>
+  <data name="field_number_of_talks" xml:space="preserve">
+    <value>Konuşma sayısı</value>
+  </data>
+  <data name="field_attendees_of_talks" xml:space="preserve">
+    <value>Konuşmaların katılımcı sayısı</value>
+  </data>
+  <data name="field_number_of_followers" xml:space="preserve">
+    <value>Takipçi sayısı</value>
+  </data>
+  <data name="field_number_of_videos" xml:space="preserve">
+    <value>Video sayısı</value>
+  </data>
+  <data name="field_number_of_events" xml:space="preserve">
+    <value>Organizasyon sayısı</value>
+  </data>
+  <data name="onboarding_1_title" xml:space="preserve">
+    <value>Basit ve etkili</value>
+  </data>
+  <data name="onboarding_1_description" xml:space="preserve">
+    <value>Tüm topluluk etkinliklerinizi avuç içinizden yönetin.</value>
+  </data>
+  <data name="onboarding_3_title" xml:space="preserve">
+    <value>Hızlı Ekle</value>
+  </data>
+  <data name="onboarding_3_description" xml:space="preserve">
+    <value>Kopyaladığınız URL'leri doğruca katkıya çevirin. Katkılara ilişkin erişebildiğimiz tüm bilgileri sizin için dolduracağız.</value>
+  </data>
+  <data name="language_turkish" xml:space="preserve">
+    <value>Türkçe</value>
+  </data>
+  <data name="about_credits_description" xml:space="preserve">
+    <value>Aşağıda listelenen harika insanlar olmasaydı bu uygulama mümkün olamazdı. Bu nedenle katkılarını kutlamak için burada bir yer kazandılar.</value>
+  </data>
+  <data name="about_credits_title" xml:space="preserve">
+    <value>Emeği Geçenler</value>
+  </data>
+  <data name="acknowledgementtype_tester" xml:space="preserve">
+    <value>Tester</value>
+  </data>
+  <data name="acknowledgementtype_translator" xml:space="preserve">
+    <value>Çevirmen</value>
+  </data>
+  <data name="sign_in" xml:space="preserve">
+    <value>Sign in</value>
```

Comment:
@cihanyakar I already added the 3 English versions of the labels here for you :)